lib/mandrill_dm/message.rb

```diff
@@ -276,6 +276,15 @@ def inline_attachments?
       mail.attachments.any?(&:inline?)
     end
 
+    def html_content?
+      return true if mail.content_type.nil?
+      mail.content_type =~ %r{text/html} ? true : false
```

Comment:
@kshnurov @Sysqa @febeling It looks like content-type can be an exact match where it returns `text/html` or in the case where there is a charset, the content-type should be delimited with a semi-colon and be the beginning of the return value, so something like this should cover both cases? `mail.content_type ~= /^text\/html[;]?.*/`